### ISSUE TITLE
Possible flipped if statement in stopSync method

### ISSUE THREAD

**Issue by mdwekat** [iyobo/pouchorm#19]
I noticed that in the source the following method

```typescript
static stopSync(fromDB: string, toDB?: string) {
    if (toDB)
      Object.values(PouchORM.activeSyncOperations[fromDB] || {}).forEach(it => it.cancel());
    else
      PouchORM.activeSyncOperations[fromDB]?.[toDB]?.cancel();
}
```

I think the if statement in this situation is flipped and should be

```typescript
static stopSync(fromDB: string, toDB?: string) {
    if (!toDB) // to enter the block if toDB is not defined
      Object.values(PouchORM.activeSyncOperations[fromDB] || {}).forEach(it => it.cancel());
    else
      PouchORM.activeSyncOperations[fromDB]?.[toDB]?.cancel();
}
```

**Comment by iyobo** [comment]
@mdwekat You are correct!

**Comment by iyobo** [comment]
Fixed in  pouchorm@2.1.2
